app/modules/ads/bill-the-lizard.js

```diff
@@ -0,0 +1,91 @@
+import targeting from './targeting';
+
+const cheshirecatPredictions = {};
+
+let cheshirecatConfig = null;
+let cheshirecatCalled = false;
+let biddersCalled = false;
+
+export default {
+  configureBillTheLizard(instantGlobals) {
+    const { context, slotService } = window.Wikia.adEngine;
+    const { billTheLizard } = window.Wikia.adServices;
+
+    if (context.get('bidders.prebid.bidsRefreshing.enabled')) {
+      cheshirecatConfig = instantGlobals.wgAdDriverBillTheLizardConfig || {};
+
+      context.set('services.billTheLizard.projects', cheshirecatConfig.projects);
+      context.set('services.billTheLizard.timeout', cheshirecatConfig.timeout || 0);
+
+      billTheLizard.projectsHandler.enable('cheshirecat');
```

Comment:
What about enabling it only when `bidders.prebid.bidsRefreshing.enabled` and repeatable boxads are enabled?